Logo/Logo files

```diff
@@ -0,0 +1 @@
+
```

Comment:
what is this file?

Reply:
I dont know what file is that, can you see the images of the logos? I will rename all the logos as you want. Greetings